src/app/modules/core/advertisement/components/card/card-bottom/review-dialog/review-dialog.component.html

```diff
@@ -0,0 +1,5 @@
+<h2 mat-dialog-title>All Reviews</h2>
```

Comment:
@Gabriele-bil al momento non ci sono tutte queste reviews nei dati mockati, ma nell'ipotesi che ce ne fossero parecchie in questo modo la visualizzazione si bloccherebbe alle prime X (in base alla grandezza del monitor)

Secondo me si potrebbe impostare un'altezza massima del dialog che si va a creare così se le reviews sono parecchie verranno visualizzate tutte e basterà scorrere all'interno del dialog per visualizzarle tutte

Questo è quello che si vede se ad esempio si mettono 10 reviews
<img width="1680" alt="Schermata 2020-11-28 alle 20 54 43" src="https://user-images.githubusercontent.com/67282965/100524911-600cdb00-31bc-11eb-8ce1-12638aedc7ed.png">

Questo invece quello che si vedrebbe mettendo un div e dando un altezza fissa
<img width="1680" alt="Schermata 2020-11-28 alle 21 00 46" src="https://user-images.githubusercontent.com/67282965/100524944-e45f5e00-31bc-11eb-95d5-7907af1d24b2.png">

Reply:
Hai ragione, effettivamente non avevo provato con tutte queste reviews, cambio subito 👍